### PR TITLE
Add parser - foreman-ssl_access_ssl.log

### PR DESCRIPTION
**Pull request by sagaraivale**
The parser used to parse the content of following paths:
1.```var/log/httpd/foreman-ssl_access_ssl.log```
2.```sos_commands/foreman/foreman-debug/var/log/httpd/foreman-ssl_access_ssl.log```

### CODE REVIEW

**Comment by sagaraivale** [comment]
@xiangce @vishwanathjadhav : Kindly review the parser.

File: insights/config/specs.py [hunk 1]
```diff
@@ -239,9 +241,9 @@
     "modinfo"                   : CommandSpec("/usr/sbin/modinfo {module}", module=r"\S+"),
     "modprobe.conf"             : PatternSpec(r"etc/modprobe\.conf"),
     "modprobe.d"                : PatternSpec(r"etc/modprobe.d/.*\.conf"),
-    "mongod_conf"               : First([PatternSpec("etc/mongod.conf"),
-                                    PatternSpec("etc/mongodb.conf")]),
-    "rh_mongodb26_conf"         : PatternSpec("etc/opt/rh/rh-mongodb26/mongod.conf"),
+    "mongod_conf"               : All([SimpleFileSpec("etc/mongod.conf"),
+                                    SimpleFileSpec("etc/mongodb.conf"),
+                                    SimpleFileSpec("etc/opt/rh/rh-mongodb26/mongod.conf")]),
```
Comment (xiangce) [inline insights/config/specs.py]: These modifications shouldn't be here in your PR, you might need to rebase your branch to the 1.x.

File: insights/parsers/foreman_log.py [hunk 2]
```diff
@@ -69,17 +69,16 @@ class CandlepinErrorLog(LogFileOutput):
         2016-09-07 15:20:33,796 [=, org=] WARN  org.apache.qpid.transport.network.security.ssl.SSLUtil - Exception received while trying to verify hostname
         2016-09-07 15:27:34,367 [=, org=] WARN  org.apache.qpid.transport.network.security.ssl.SSLUtil - Exception received while trying to verify hostname
         2016-09-07 16:49:24,650 [=, org=] WARN  org.apache.qpid.transport.network.security.ssl.SSLUtil - Exception received while trying to verify hostname
-        2016-09-07 18:07:53,688 [req=d9dc3cfd-abf7-485e-b1eb-e1e28e4b0f28, org=org_ray] ERROR org.candlepin.sync.Importer - Conflicts occurred during import that were
+        2016-09-07 18:07:53,688 [req=d9dc3cfd-abf7-485e-b1eb-e1e28e4b0f28, org=org_ray] ERROR org.candlepin.sync.Importer - Conflicts occurred during hat were
```
Comment (xiangce) [inline insights/parsers/foreman_log.py]: Please confirm this too.

File: insights/parsers/foreman_log.py [hunk 3]
```diff
@@ -69,17 +69,16 @@ class CandlepinErrorLog(LogFileOutput):
...
 
     Examples:
         >>> candlepin_log = shared[Candlepin_Error_Log]
         >>> candlepin_log.get('req=d9dc3cfd-abf7-485e-b1eb-e1e28e4b0f28')
-        ['2016-09-07 18:07:53,688 [req=d9dc3cfd-abf7-485e-b1eb-e1e28e4b0f28, org=org_ray] ERROR org.candlepin.sync.Importer - Conflicts occurred during import that were
-        not overridden:', '2016-09-07 18:07:53,690 [req=d9dc3cfd-abf7-485e-b1eb-e1e28e4b0f28, org=org_ray] ERROR org.candlepin.sync.Importer - [DISTRIBUTOR_CONFLICT]',
-        '2016-09-07 18: 07:53,711 [req=d9dc3cfd-abf7-485e-b1eb-e1e28e4b0f28, org=org_ray] ERROR org.candlepin.resource.OwnerResource - Recording import failure
+        ['2016-09-07 18:07:53,688 [req=d9dc3cfd-abf7-485e-b1eb-e1e28e4b0f28, org=org_ray] ERROR org.candlepin.sync.Importer - Conflicts occurred during import that were not overridden:', '2016-09-07 18:07:53,690 [req=d9dc3cfd-abf7-485e-b1eb-e1e28e4b0f28, org=org_ray] ERROR org.candlepin.sync.Importer - [DISTRIBUTOR_CONFLICT]',
+       '2016-09-07 18: 07:53,711 [req=d9dc3cfd-abf7-485e-b1eb-e1e28e4b0f28, org=org_ray] ERROR org.candlepin.resource.OwnerResource - Recording import failure
```
Comment (xiangce) [inline insights/parsers/foreman_log.py]: And this part.

**Comment by sagaraivale** [comment]
@xiangce  : I have made changes as per suggestions. Kindly review it.

**Comment by xiangce** [comment]
Looks good to me. Please resubmit it to the master branch, I will merge them together.

**Pull request merged**